packages/askui-nodejs/src/utils/http/http-client-got.ts

```diff
@@ -15,45 +15,68 @@ export class HttpClientGot {
     readonly token?: string,
     readonly customHeaders?: Record<string, string>,
     private readonly cookies: Record<string, string> = {},
-    readonly proxyAgents?: { http: http.Agent, https: https.Agent },
+    readonly proxyAgents?: { http: http.Agent; https: https.Agent },
```

Comment:
Why did you switched to a semicolon?

Reply:
Was my formatter... Currently, we do not enforce this automatically so I would keep it this way until we do...